next.config.js

```diff
@@ -6,10 +6,14 @@ const withBundleAnalyzer = require("@next/bundle-analyzer")({
 const nextConfig = {
   poweredByHeader: false,
 
+  // Public resources are loaded with the `router.basePath` to
+  // support deployments to sub-paths of a domain.
+  basePath: "",
```

Comment:
This is not strictly necessary, because the `basePath` defaults to an empty string anyway. However, its more obvious if we declare it anyway.